frontend/src/Blips/EditForm/styles.css

```diff
@@ -0,0 +1,630 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    text-rendering: optimizeLegibility;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-tap-highlight-color: transparent;
+  }
+  
+  html {
+    font-size: 16px;
+    font-family: -apple-system, BlinkMacSystemFont, "avenir next", avenir,
+      "helvetica neue", helvetica, ubuntu, roboto, noto, "segoe ui", arial,
+      sans-serif;
+    margin: 0;
+    padding: 0;
+    text-decoration-skip-ink: "auto";
+  }
+  
+  :root {

```

Comment:
this is a general style that should belong to a layout or index.css
also, why is this needed?